docs/framework/data/adonet/ef/language-reference/intersect-entity-sql.md

```diff
@@ -29,7 +29,7 @@ expression INTERSECT expression
   
 2. Pass the following query as an argument to the `ExecuteStructuralTypeQuery` method:  
   
- [!code-csharp[DP EntityServices Concepts 2#INTERSECT](../../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts 2/cs/entitysql.cs#intersect)]  
+ [!code-sql[DP EntityServices Concepts 2#INTERSECT](../../../../../../samples/snippets/csharp/VS_Snippets_Data/dp entityservices concepts 2/cs/entitysql.cs#intersect)]  
```

Comment:
You'll also need to change the path for all these examples.

```suggestion
 [!code-sql[DP EntityServices Concepts 2#INTERSECT](~/samples/snippets/tsql/VS_Snippets_Data/dp entityservices concepts 2/tsql/entitysql.cs#intersect)]  
```

Build doesn't say anything but they are blank as is:
![image](https://user-images.githubusercontent.com/12971179/66022972-7c23c800-e4a4-11e9-93d8-4b78f3ca627c.png)

Reply:
Fixed the example paths so that they now point to the SQL examples.